.circleci/config.yml

```diff
@@ -99,13 +103,33 @@ jobs:
           command: "prove -v . |& tee ../nginx-plus-module-prove-test-verbose || true"
           environment:
             TEST_NGINX_BINARY: "/usr/sbin/nginx"
-            TEST_NGINX_GLOBALS: "load_module /tmp/workspace/ubuntu2204/<<parameters.version>>/ngx_http_akita_module.so;"
+            TEST_NGINX_GLOBALS: "load_module /tmp/workspace/ubuntu_2204/<<parameters.version>>/ngx_http_akita_module.so;"
             TEST_NGINX_GLOBALS_HTTP: "akita_agent localhost:5555; akita_enable off;"
           working_directory: "nginx-tests-tip"
       - store_artifacts:
           path: nginx-plus-module-prove-test-verbose
           destination: "nginx-test-results-<<parameters.version>>/nginx-plus-module-prove-test-verbose"
 
+  release-module:
+    docker:
+      - image: cimg/go:1.20
+    resource_class: medium
+    steps:
+      - checkout:
+          path: "akita"      
+      - attach_workspace:
+          at: ./artifacts
+      - run:
+          name: "Build GHR"
+          command: go install github.com/tcnksm/ghr@latest
+      - run:
+          # Fails if release already exists; bump VERSION file on the release branch in order to kick off a new release.
+          name: "Create release"
+          command: |
+            VERSION=`cat akita/VERSION`
```

Comment:
backticks are generally discouraged these days, safer to use command substitution syntax eg `VERSION="$(cat akita/VERSION)"`

Reply:
Changed in https://github.com/akitasoftware/akita-nginx-module/pull/19/commits/dbc766aee666e4320baf3843a055fd1a283f4252